proto/node.proto

```diff
@@ -51,7 +51,12 @@ service DealManagement {
     // Status produces a detailed info about deal with given ID.
     rpc Status(BigInt) returns (DealInfoReply) {}
     // Finish finishes a deal with given ID
+    // Deprecated: use FinishDeals instead
```

Comment:
I don't think we should deprecate non-pluralized method.

Reply:
it's functionality is dublicated, plural methods are quite easy to use and do the same thing if you pass single value. I still think we should deprecate them.